src/mutation.ts

```diff
@@ -242,7 +285,8 @@ export class Mutation {
    * ]);
    * @private
    */
-  static encodeDelete(data?) {
+  static encodeDelete(data?: Metadata): EncodeDeleteDataResponse
```

Comment:
Usually `encode` methods are just formatting request objects. I think you could use [`IDeleteFromColumn`](https://github.com/googleapis/nodejs-bigtable/blob/master/proto/bigtable.d.ts#L3336) here, but moving forward I think we should avoid using `Response` for these kinds of methods.

Reply:
Just curious, want to learn this right - is there something specific about `static` methods that mean I should avoid a `Response` return annotation or is it something more general about `Response`s?

Reply:
More general for `Response`, IMO we should only use that word to describe actual response payloads, where this method I think is just something we use internally to transform an object into the expected proto format.

Reply:
Makes sense, cool. Noted 👍